text/src/autogluon/text/automm/predictor.py

```diff
@@ -922,6 +937,13 @@ def load(
         path = os.path.expanduser(path)
         assert os.path.isdir(path), f"'{path}' must be an existing directory."
         config = OmegaConf.load(os.path.join(path, "config.yaml"))
+
+        for model_name in config.model.names:
+            if model_name == "clip" or "hf_text" in model_name:
+                model_config = getattr(config.model,model_name)
+                if 'LOCAL:' in model_config.checkpoint_name:
+                    model_config.checkpoint_name = os.path.join(path,model_config.checkpoint_name.split('/')[1])
```

Comment:
Change to `model_config.checkpoint_name = os.path.join(path, model_config.checkpoint_name[len('local://'):])`